recipe/meta.yaml

```
@@ -155,6 +154,7 @@ outputs:
         - krb5                               # [not win]
         - libblas
         - liblapack
+        - sed                                # [linux]
```

To avoid regressing again:
```suggestion
        # sed is in host env because its path is recorded in R's shell wrapper
        - sed                                # [linux]
```